eva_submission/retrieve_eload_and_project_from_lts.py

```diff
@@ -0,0 +1,135 @@
+import fileinput
+import logging
+import os
+
+import yaml
+from ebi_eva_common_pyutils import command_utils
+from ebi_eva_common_pyutils.config import cfg
+from ebi_eva_common_pyutils.logger import logging_config as log_cfg
+
+logger = log_cfg.get_logger(__name__)
+
+
+class ELOADRetrieval():
+
+    def create_dir_if_not_exist(self, dir_path):
+        if not os.path.exists(dir_path):
+            os.makedirs(dir_path)
+
+    def get_compressed_files_in_dirs(self, dir_path):
+        # To uncompress - retrieve every compressed file from the directory (including subdirectories)
+        # but exclude compressed vcf files (the ones ending with .vcf.gz)
+        if os.path.exists(dir_path):
+            file_list = []
+            for root, directories, files in os.walk(dir_path):
+                for file in files:
+                    if file[-3:] == '.gz' and file[-7:] != '.vcf.gz':
```

Comment:
@tcezard I am guessing this check is sufficient. should we expect to find any "double-gzipped" files inside the directories?